aspnetcore/grpc/interprocess.md

```diff
@@ -11,11 +11,36 @@ uid: grpc/interprocess
 
 By [James Newton-King](https://twitter.com/jamesnk)
 
-gRPC calls between a client and service are usually sent over TCP sockets. TCP was designed for communicating across a network. [Inter-process communication (IPC)](https://wikipedia.org/wiki/Inter-process_communication) is more efficient than TCP when the client and service are on the same machine. This document explains how to use gRPC with custom transports in IPC scenarios.
+Apps on the same machine can be designed to communicate with each other. Operating systems provide technologies for enabling fast and efficient [inter-process communication (IPC)](https://wikipedia.org/wiki/Inter-process_communication) between apps. Popular examples of IPC technologies are named pipes (Windows only), and unix domain sockets (Windows, Linux and MacOS).
 
-## Server configuration
+.NET provides excellent support for inter-process communication using gRPC.
 
-Custom transports are supported by [Kestrel](xref:fundamentals/servers/kestrel). Kestrel is configured in `Program.cs`:
+## Getting started with gRPC
+
+gRPC calls are always sent from a client to a server. To communicate between apps on a machine with gRPC, at least one app must host an ASP.NET Core gRPC server.
+
+ASP.NET Core and gRPC can be hosted in any app using .NET Core 3, or .NET 5 or later by adding the `Microsoft.AspNetCore.App` framework to the project.
```

Comment:
The commas/conjunctions are challenging. Two idears here ...

This one assumes that everyone knows that 4.x isn't part of the sequence, and we use this convention regularly around the docs without any fuss from readers ...

```suggestion
ASP.NET Core and gRPC can be hosted in any app using .NET Core 3.1 or later by adding the `Microsoft.AspNetCore.App` framework to the project.
```

This one lists them to avoid the comma/conjunction problem ...

```suggestion
ASP.NET Core and gRPC can be hosted by adding the `Microsoft.AspNetCore.App` framework to the project for the following targeted frameworks:

* .NET Core 3.1
* .NET 5.0 or later
```